build.gradle.kts

```diff
@@ -27,6 +27,7 @@ buildscript {
         classpath(libs.hilt.gradlePlugin)
         val fdroidBuild = (System.getenv("flavor")
             ?: System.getenv("FLAVOR")
+            ?: System.getenv("CUSTOM_FLAVOR")
```

Comment:
We should really document these in a `BUILDING.md` file or similar.

So we could summarise all of the process not only for newcomers but for us. There might be room to simplify stuff but I can say from my side that it's a bit hard to wrap my head around all the env variables needed to make releases.